packages/scss/src/components/menu/component.scss

```diff
@@ -43,12 +43,13 @@
 			padding: var(--components-menu-padding);
 			border-radius: var(--commons-borderRadius-M);
 			color: var(--palettes-grey-700);
+			display: inline-flex;
+			align-items: center;
+			gap: var(--spacings-XS);
 			transition-duration: var(--commons-animations-durations-fast);
 			transition-property: color;
 			position: relative;
 			text-decoration: none;
-			text-align: center;
```

Comment:
Central alignment of the text can be useful when the contents pass on two lines.